UI/window-basic-auto-config.cpp

```diff
@@ -566,7 +566,8 @@ void AutoConfigStreamPage::UpdateKeyLink()
 	if (serviceName == "Twitch") {
 		streamKeyLink =
 			"https://www.twitch.tv/broadcast/dashboard/streamkey";
-	} else if (serviceName == "YouTube / YouTube Gaming") {
+	} else if ((serviceName == "YouTube - RTMP") or
```

Comment:
Checking the codebase, I think the convention is using || instead of the keyword or. And same below.